EIPS/eip-1679.md

```diff
@@ -28,12 +28,10 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 - [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs
 - [EIP-1344](https://eips.ethereum.org/EIPS/eip-1344): Add ChainID opcode
 - [EIP-2028](https://eips.ethereum.org/EIPS/eip-2028): Calldata gas cost reduction
-
-### Tentatively Accepted EIPs
-#### Istanbul 1
 - [EIP-1884](https://eips.ethereum.org/EIPS/eip-1884): Repricing for trie-size-dependent opcodes
```

Comment:
It would be nice for them to be ordered numerically, i.e. `1344` -> `1884` -> `2028` -> `2200`.

Reply:
Updated